server/server.go

```diff
@@ -3593,3 +3599,42 @@ func (s *Server) updateRemoteSubscription(acc *Account, sub *subscription, delta
 
 	s.updateLeafNodes(acc, sub, delta)
 }
+
+func (s *Server) startRateLimitLogExpiration() {
+	s.mu.Lock()
+	interval := int64(time.Second)
+	s.rateLimitLoggingCh = make(chan int64, 1)
```

Comment:
Why not just make a chan on time.Duration to avoid casts?

Reply:
If I do that, then the cast will have to be done when checking if it should be expired:
```
if start := v.(int64); now-start >= interval {
```
would have to be `int64(interval)`. I originally had actual duration and then do `time.Since(start)` (by setting in the map time.Now() instead of time.Now().UnixNano(), but what I did not like is the idea that time.Since() will then evaluate the "time now" at every call?  Maybe that's not a big deal.. what do you think?